presto-cassandra/src/test/java/com/facebook/presto/cassandra/CassandraTestingUtils.java

```diff
@@ -44,7 +44,7 @@ private CassandraTestingUtils() {}
     public static void createTestTables(Cluster cluster, String keyspace, Date date)
     {
         try (Session session = cluster.connect()) {
-            createOrReplaceKeyspace(session, keyspace);
+            createKeyspace(session, keyspace);
         }
         try (Session session = cluster.connect(keyspace)) {
```

Comment:
It is propagated. If it fails when creating a keyspace it will throw an exception.